app/backend/db/schemas.py

```diff
@@ -66,8 +66,8 @@ class PostDateDataBody(BaseModel):
     end_date: date = None
     state: AnswerStateTypes
     reason: str = None
-    reported_by_id: int
-    reported_time: datetime
+    # reported_by_id: int
+    # reported_time: datetime
```

Comment:
if U need it, remove the comment 
else remove the #

Reply:
.